config.toml

```diff
@@ -160,34 +160,34 @@ js = [
 fullversion = "v1.19.0"
 version = "v1.19"
 githubbranch = "v1.19.0"
-docsbranch = "release-1.19"
+docsbranch = "master"
 url = "https://kubernetes.io"
 
 [[params.versions]]
-fullversion = "v1.18.0"
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "v1.18.0"
+githubbranch = "v1.18.8"
 docsbranch = "release-1.18"
-url = "https://v-18.docs.kubernetes.io"
+url = "https://v-1.18.docs.kubernetes.io"
```

Comment:
```suggestion
url = "https://v1-18.docs.kubernetes.io"
```

?

Reply:
Thank you for catching it @justaugustus . This is like the 4th time this week I am masking mistakes on this particular url. Fix coming up right away

Reply:
Fixed it